modules/cnv-creating-vm.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-create-vms.adoc
+
+[id="cnv-creating-vm_{context}"]
+= Creating a virtual machine from the CLI
+
+The `spec` object of the VirtualMachine configuration file references
+the virtual machine settings, such as the number of cores and the amount
+of memory, the disk type, and the volumes to use.
+
+Attach the virtual machine disk to the virtual machine by referencing
+the relevant PVC `claimName` as a volume.
+
+[NOTE]
+====
+https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/[ReplicaSet]
```

Comment:
Can I have a bit more context?

Reply:
Changed Note to read:
"A  ReplicaSet is often used to guarantee the availability of a specified number of identical Pods.
ReplicaSet is not currently supported in {ProductName}."